packages/rollup-tests/package.json

```diff
@@ -3,7 +3,7 @@
   "private": true,
   "scripts": {
     "test": "mocha --file ./src/intercept/check.js test/test.js",
-    "test--update": "mocha --jobs 1 --file ./src/intercept/update-test-status.js test/test.js",
+    "test-u": "mocha --jobs 1 --file ./src/intercept/update-test-status.js test/test.js",
```

Comment:
It will break `just test-node rollup --update`. We need to fix 

```
test-update:
    just test-rust
    just test-node all -u -> --update
```

Reply:
Indeed, I intended to add a `test-u` command to support both `-u` and `--update`, but it seems I forgot to push it.